readme.md

```diff
@@ -1,8 +1,20 @@
-# wordpress-micropub [![Circle CI](https://circleci.com/gh/snarfed/wordpress-micropub.svg?style=svg)](https://circleci.com/gh/snarfed/wordpress-micropub)
```

Comment:
this should be preserved. if it can go into readme.txt and look right in the wordpress plugin directory, i'm all for that!

Reply:
I'll do a PR to reconcile the old and new readmes. But the goal was that both be versions of the same file. They effectively were, but the conversion was copied by hand.

Reply:
understood! it's a worthy goal. let's reconcile them in this PR, so we know it's possible with the automated script, then i'll be on board.